lib/matplotlib/axes/_base.py

```diff
@@ -1093,6 +1081,13 @@ def cla(self):
             self.yaxis.set_visible(yaxis_visible)
             self.patch.set_visible(patch_visible)
 
+        # It is not clear to me (EF) why this reset is needed here, but
```

Comment:
`Axes`, `Axis`, `Spine`, and `Tick` are all inexplicably intertwined. I've been trying to disconnect them (e.g., #8678), but there's always one last bug every time...